crates/solidity/inputs/schema/topics/01-file-structure/05-trivia.yml

```diff
@@ -47,7 +47,7 @@
   kind: Token
   delimitedBy:
     open: /*
-    config: { name: content }
+    config: { name: Content }
```

Comment:
I will add a TODO for myself to add validation for these names. :+1

Reply:
thanks